### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by copocaneta**
## PR Checklist
Please check if your PR fulfills the following requirements:

- [X] The commit message follows our guidelines: https://github.com/blockcoders/nestjs-ethers/blob/master/CONTRIBUTING.md
- [X] Tests for the changes have been added (for bug fixes / features)
- [X] Docs have been added / updated (for bug fixes / features)

## PR Type
What kind of change does this PR introduce?

- [ ] Bugfix
- [ ] Feature
- [ ] Code style update (formatting, local variables)
- [ ] Refactoring (no functional changes, no api changes)
- [ ] Build related changes
- [ ] CI related changes
- [X] Other... Please describe: 

README, documentation improvement/update.

## What is the current behavior?

`this.ethersSigner` does not exist on this class. It should be the `this.signer` from the constructor, right?

## What is the new behavior?
It should be the `this.signer` from the constructor, right?

## Does this PR introduce a breaking change?

- [ ] Yes
- [X] No

## Other information
Nothing else.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -472,7 +472,7 @@ export class TestService {
     private readonly signer: EthersSigner,
```
Comment (vanb) [inline README.md:472]: Alternatively, could rename this variable to `ethersSigner` to be consistent with other examples.
```suggestion
    private readonly ethersSigner: EthersSigner,
```

Comment (copocaneta) [reply README.md]: Perfect. Done.

**Comment by copocaneta** [comment]
Renamed `signer` variable to `ethersSigner` to be consistent with other examples

**Review by GustavoRSSilva — APPROVED** [review]

**Review by vanb — APPROVED** [review]

**Review by 0xslipk — APPROVED** [review]
LGTM. Thank you

**Pull request merged**